src/settings-ui/Settings.UI/OOBE/Enums/PowerToysModulesEnum.cs

```diff
@@ -7,6 +7,7 @@ namespace Microsoft.PowerToys.Settings.UI.OOBE.Enums
     public enum PowerToysModulesEnum
     {
         Overview = 0,
+        AlwaysOnTop,
```

Comment:
where is this enum being leveraged.  Inserting a new entry at the top could have rippling effects since this could be treated as an int

Reply:
It's used for centralized initialization of the oobe pages and keeping them in order. So if I added it at the end, the AOT page will be the last.

https://github.com/microsoft/PowerToys/blob/c1a80a20016788b24a387a71936f5ff70290ca07/src/settings-ui/Settings.UI/OOBE/Views/OobeShellPage.xaml.cs#L74-L85

https://github.com/microsoft/PowerToys/blob/c1a80a20016788b24a387a71936f5ff70290ca07/src/settings-ui/Settings.UI/OOBE/Views/OobeAwake.xaml.cs#L20